scripts/2.mutation-classifier.py

```diff
@@ -36,8 +38,28 @@
 
 # In[3]:
 
-# We're going to be building a 'TP53' classifier 
-mutation_id = '7157' # TP53
+
+# We're going to be building a classifier with multiple genes filtered by two diseases 
+# Example:
+# gene_ids = ['7157', '7158', '7159', '7161']
+
+# Information passed into the notebook is stored in environment variables
+gene_ids = os.environ.get('gene_ids')
+if not gene_ids:
+    gene_ids = ['7157'] # TP53 is the default
+else:
+    gene_ids = gene_ids.split('-')
```

Comment:
I think comma separating makes more sense than hyphen. @dcgoss how hard would it to switch to commas?